index.html

```diff
@@ -1059,24 +1213,35 @@ <h2>Evidence</h2>
 }
         </pre>
 
+        <p class="note">
+For the avoidance of doubt, the <em>evidence</em> <a>property</a> provides
+information that is different from and complementary to the
+<a href="#proofs-aka-signatures">proof</a> property. The <em>evidence</em>
+property is used to express supporting information, such as documentary
+evidence, related to the integrity of the credential. In contrast, the
+<em>proof</em> property is used to express machine-verifiable mathematical
+proofs related to the authenticity of the isuer and integrity of the credential.
```

Comment:
typo "isuer" => "issuer"

Reply:
Fixed in 94904807a4f23cc7922d44de9f9c6719eddb2b75.